### PR TITLE
coc: suggest rewording of reverse-ism clause

### PR DESCRIPTION
**Pull request by zkat**
A while back, a conversation about this CoC came up over at https://github.com/videojs/video.js/pull/4160 when a PR for it was made. I spent some time thinking about it, and I'm wondering if it *is* worth rewording these bits. I'm hoping for some discussion and feedback on whether this is worth changing, whether the new edit is good, or whether the entire premise of changing it is basically preposterous: I don't want to neglect this particular aspect from this CoC, which is _meant_ to be thorough so that everyone knows the ground rules.

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ Examples of unacceptable behavior by participants include:
   * Publishing private screenshots or quotes of interactions in the context of this project without all quoted users' *explicit* consent.
   * Publishing of private communication that doesn't have to do with reporting harrassment.
   * Any of the above even when [presented as "ironic" or "joking"](https://en.wikipedia.org/wiki/Hipster_racism).
-  * Any attempt to present "reverse-ism" versions of the above as violations. Examples of reverse-isms are "reverse racism", "reverse sexism", "heterophobia", and "cisphobia".
+  * Any attempt to deflect unacceptable behavior by using "reverse-isms", or equating "reverse-isms" with oppressive speech about a minority group (which can be an attempt to minimize the legacy and impact of long-standing and systemic oppression of minorities). Examples of reverse-isms are "reverse racism", "reverse sexism", "heterophobia", and "cisphobia". This project recognizes oppression as directional and will adjust its perspective accordingly.
```
Comment (AgentAntelope) [inline CODE_OF_CONDUCT.md]: > attempt to deflect unacceptable behavior

I don't think this is enough to clarify that the act of presenting a "reverse-ism" is itself unacceptable

**Review by AgentAntelope — CHANGES_REQUESTED** [review]

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (zkat) [reply CODE_OF_CONDUCT.md]: Do you have any suggested rewording, or do you feel this entire PR goes counter to what you think is best?

Comment (AgentAntelope) [reply CODE_OF_CONDUCT.md]: I'd say that:
> This project recognizes oppression as directional and will adjust its perspective accordingly.

is a valuable and important sentence; but the problem isn't just people invoking "reverse-isms" as a comparator to existing structural oppression it's also people saying things like "You're racist against white people".

Comment (othiym23) [reply CODE_OF_CONDUCT.md]: I agree with @AgentAntelope. 

I like the attempt to provide context in the PR, but I also like the directness of the original text. That said, if you're going to clarify things, anyone who "has a tough time" "understanding" what a "reverse -ism" is is probably also going to have difficulty "understanding" what "oppression as directional" means. (Sorry for all the "quotes", but... you know what I mean.)

I think it's worth making a change, but the current proposal isn't as clear as the original. Ganbare!

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ When reporting, please include any relevant details, links, screenshots, context
 
 ## Our Pledge
 
-In the interest of fostering an open and welcoming environment, we as contributors and maintainers of this project pledge to making participation in our community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, gender identity and expression, level of experience, technical preferences, nationality, personal appearance, race, religion, or sexual identity and orientation.
+In the interest of fostering an open and welcoming environment, we as contributors and maintainers of this project pledge to making participation in our community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, gender identity and expression, level of experience, technical preferences, nationality, personal appearance, race, religion, or sexual identity and orientation. This project further recognizes oppression as directional and complex and will adjust its perspective on issues accordingly.
```
Comment (cjwilburn) [inline CODE_OF_CONDUCT.md:19]: I'm a tad late to the party, but wondering a bit about the `oppression is directional` part. I understand what it means, and I think I know what you're trying to get across by explicitly stating it, but I'm not sure if this is needed.

File: CODE_OF_CONDUCT.md [hunk 3]
```diff
@@ -37,7 +37,7 @@ Examples of unacceptable behavior by participants include:
   * Publishing private screenshots or quotes of interactions in the context of this project without all quoted users' *explicit* consent.
   * Publishing of private communication that doesn't have to do with reporting harrassment.
   * Any of the above even when [presented as "ironic" or "joking"](https://en.wikipedia.org/wiki/Hipster_racism).
-  * Any attempt to present "reverse-ism" versions of the above as violations. Examples of reverse-isms are "reverse racism", "reverse sexism", "heterophobia", and "cisphobia".
+  * Any attempt to present "reverse-ism" versions of the above as violations. Examples of reverse-isms are "reverse racism", "reverse sexism", "heterophobia", and "cisphobia". Aggressive/angry language related to this has its place, though, and might be best left out of shared spaces. Try to be kind.
```
Comment (cjwilburn) [inline CODE_OF_CONDUCT.md:40]: Is this around `bigotry` versus `racism`?

**Review by cjwilburn — COMMENTED** [review]
If these questions address issues that you've already discussed, feel free to disregard.

**Review by marvel3283 — APPROVED** [review]
